rspotify-http/src/reqwest.rs

```diff
@@ -67,6 +70,17 @@ impl Default for ReqwestClient {
     }
 }
 
+#[cfg(target_arch = "wasm32")]
+impl Default for ReqwestClient {
+    fn default() -> Self {
```

Comment:
reqwest doesn't implement the `timeout` function when built for wasm, hence the need for a separate implementation.